ssh/tailssh/tailssh.go

```diff
@@ -400,7 +446,7 @@ func (srv *server) newConn() (*conn, error) {
 		// Stop accepting new connections.
 		// Connections in the auth phase are handled in handleConnPostSSHAuth.
 		// Existing sessions are terminated by Shutdown.
-		return nil, errDenied("tailscale: server is shutting down")
+		return nil, errors.New("server is shutting down")
```

Comment:
Drive-by: there was no point to returning `errDenied` here since we don't even have an SSH connection at this point.